account-kit/react/src/hooks/useAuthenticate.ts

```diff
@@ -48,6 +53,10 @@ export function useAuthenticate(
   const { queryClient } = useAlchemyAccountContext();
   const signer = useSigner();
 
+  const {
+    authStep: { type: stage },
+  } = useAuthContext();
```

Comment:
Oh I just saw this, so this will cause problems for anyone not using the UI components. Most of the hooks should work with or without the react components, so we probably don't want to include this call here

Reply:
Ok we can remove  and use useAuthContext directly in the auth card wrapper.  The intention behind this was to keep the interface cleaner by having useAuthenticate handle the auth step but I did not consider consumers of useAuthenticate wouldn't also be using authContext.

Reply:
Sounds good! It has been removed